src/info/channel.go.tmpl

```diff
@@ -11,21 +11,29 @@
 	{{ $channel = $args.Get 0 }}
 {{ end }}
 
-{{ $types := cslice "Text" "DM" "Voice" "Group DM" "Category" "News" "Store" }}
+{{ $types := sdict 
+	"0" "Text" "1" "DM" "2" "Voice" "3" "Group DM" "4" "Category" "5" "Announcement" 
+	"10" "Thread - Announcement" "11" "Thread - Public" "12" "Thread - Private" 
+	"13" "Stage" "14" "Directory" "15" "Forum" "16" "Media"
```

Comment:
Newline for each entry, please.
Semantically I think this makes more sense as a dict with int keys, even though that would require an extra conversion `toInt (json $channel.Type)`.

Reply:
Should I add a comment as to why we convert to json, then int?
It seems like a waste, but is needed to get a comparable value type that preserves the value with how `.Channel.Type` is currently returned.

Reply:
Ideally `toInt` is fixed so that `toInt .Channel.Type` gives the expected result. When that happens, we can change this to just `toInt .Channel.Type`. But for the time being, a comment would be useful.